src/main/java/org/broadinstitute/hellbender/cmdline/argumentcollections/ReferenceInputArgumentCollection.java

```diff
@@ -14,6 +14,8 @@
 
     /**
      * Get the reference file specified at the command line, creating the File object first if necessary.
+     *
+     * Use getReferencePath() instead.
```

Comment:
Why not just remove it and use wherever it appears the `getReferencePath().toFile()`?

Reply:
That's not a bad idea, but leaving it in makes it super-easy to find the places I still have to update (using the IDE's "find calls"). The latter syntax would require a full-text search, which is slower and less convenient. Also, it's public so removing it might break some code that depends on GATK. What I was thinking instead was to mark it as @deprecated once we're done cleaning house.

Reply:
That's a good point.  We should definitely fix those usages as well, but we could do it in another PR if you don't want to fit it into this one.  It looks like it needs some changes to the spark tools / one of the CNV tools.

Reply:
Dependent code on GATK should be aware that using a beta version might change the API - I think that I am one of the affected by changes in API and I am find with them, because I accepted it from the beginning...

Reply:
And I agree that it can wait, but before the release to do not have support for deprecated things...

Reply:
Good points. I went ahead and deleted getReferenceFile. The Spark usages look a bit wonky now, someone may want to revisit in a later PR and perhaps simplify the logic there.